### DISCUSSION TITLE
Event registration using fe_login and sr_feuser_register

### DISCUSSION THREAD

**Discussion by wmarie-joseph** [oliverklee/ext-seminars#2548] [Q&A]
Hello @oliverklee ,

I've been using onetimeaccount for users to login before registering to an event, it works fine, they are successfully redirected to the registration form.

The issue is that, using this method, after the user is registered if he wants to login manually, using simple login form, he doesn't have a password. i want users to be able to manage their events using the "my events" page.

I would like to replace onetimeaccount with sr_feuser_register and fe_login so the user can create an account, a password then login and register to an event. The issue i am having is that after the login, the user is not redirected to the registration form with the event that was selected before-hand.

Is there a special configuration that needs to be done for seminars to work with  sr_feuser_register and fe_login ?

**Comment by oliverklee** [comment]
As far as I know, sr_feuser_register does not support keeping the `redirect_url` GET parameter when a user creates their account. You might want to propose this feature to the author(s) of that extension, though.

**Comment by oliverklee** [comment]
(And in general, I'd recommend using `sf_register` over `sr_feuser_register` nowadays as the former is more modern and uses Fluid templates.)
